docs/whats-new/zowe-announcements.md

```diff
@@ -1,10 +1,17 @@
 # Zowe announcements
 
+## Zowe Version 3.0 technical preview
+
+Zowe publishes technical preview builds for Version 3 for all components involved. The latest versions are downloadable from the Zowe JFrog Artifactory directory - 
+[https://zowe.jfrog.io/zowe/libs-release-local/org/zowe/nightly/v3/](https://zowe.jfrog.io/zowe/libs-release-local/org/zowe/nightly/v3/). The directory contains PAX and SMP/E packages. The server-side includes the API Mediation Layer, Virtual Desktop, ZSS, and ZIS. The build is released nightly.
+
+Client-side components are also available. The CLI is available from NPM. The version starts with 8.0.0.next - [https://www.npmjs.com/package/@zowe/cli/v/8.0.0-next.202404191414](https://www.npmjs.com/package/@zowe/cli/v/8.0.0-next.202404191414).  Explorer for VSCode is available from GitHub releases - [https://github.com/zowe/zowe-explorer-vscode/releases](https://github.com/zowe/zowe-explorer-vscode/releases). Explorer for Intellij is available from the Intellij Marketplace - [https://plugins.jetbrains.com/plugin/18688-zowe-explorer](https://plugins.jetbrains.com/plugin/18688-zowe-explorer)
+
 ## Zowe Version 3.0 release delay
 
 The Zowe V3 release is postponed from its original March 28 date. The schedule for <b>Zowe V2 releases will be adjusted</b> to accommodate this change.
 
-Expect to have a <b>new V3 release date</b> and additional information on Zowe V2 releases by the end of March. Notifications will go out at that time.
+We expect to have a <b>new V3 release date</b> and additional information on Zowe V2 releases by the end of March. Notifications will go out at that time.
 
 For updates, see [vNext](https://www.zowe.org/vnext) in Zowe.org.
```

Comment:
Now that we have the Breaking Changes and Important Updates in Zowe v3 published in Zowe Docs, should we add a link to this as well? [Breaking Changes and Important Updates in Zowe v3](https://docs.zowe.org/stable/whats-new/breaking-changes-v3)